paddle/fluid/pybind/global_value_getter_setter.cc

```diff
@@ -144,44 +240,104 @@ void BindGlobalValueGetterSetter(pybind11::module *module) {
 
   py::class_<GlobalVarGetterSetterRegistry>(*module,
                                             "GlobalVarGetterSetterRegistry")
-      .def("__getitem__", &GlobalVarGetterSetterRegistry::Get)
-      .def("__setitem__", &GlobalVarGetterSetterRegistry::Set)
+      .def("__getpublic__", &GlobalVarGetterSetterRegistry::GetPublic)
```

Comment:
Why do you make "__XXX__" for these methods? Do you know these kind of naming is usually used for special python defined function/variables?

For example, if you search `__contains__, __setitem__, __getitem__`, you will know they are  python defined methods